packages/admin-stories/src/docs/components/RouterTabs/02_RouterTabs.stories.mdx

```diff
@@ -0,0 +1,36 @@
+import { Canvas, Meta, Story } from "@storybook/addon-docs";
+
+<Meta title="Docs/Components/Tabs/Router Tabs" />
+
+# Router Tabs
+
+The Comet Admin `RouterTabs` component has essentially the same functionality as `Tabs`.
+However, `RouterTabs` manages its state in the URL so that each tab has a unique URL.
+
+`RouterTabs` has the following props:
+
+-   `tabComponent` for passing a custom tab component. Per default, [Material UI's `Tab` component](https://v4.mui.com/api/tab/) is used
+-   `tabsProps` for passing props to [Material UI's `Tabs` component](https://v4.mui.com/api/tabs/)
+
+As children, an array of `RouterTab` elements should be passed to `RouterTabs`.
+Each `RouterTab` needs:
+
+-   a `label` that is shown in the tab bar
+-   a unique `path` of the tab that is attached to the URL
+-   optionally a `forceRender` flag.
```

Comment:
Maybe we could add an example where this prop should be used. I can't think of a use case immediately.

Reply:
I also couldn't think of a sensible example use case.
At first, I thought SEO purposes, but that doesn't make sense for an admin application... 🤔

Reply:
**Update:**

It's necessary for Forms that stretch over multiple Tabs because FinalForm "forgets" Fields that aren't rendered anymore. Then, form validation and dirty state don't work anymore

Reply:
@johnnyomair I added a section about the `forceRender` prop